xwiki-platform-core/xwiki-platform-tour/xwiki-platform-tour-ui/src/main/resources/TourCode/TourJS.xml

```diff
@@ -172,7 +172,9 @@ require(['jquery', 'xwiki-meta'], function ($, xm) {
     // Create the popover
     var popover = $('&lt;div class="popover-content"&gt;#tr("tour.popover.show.hint")&lt;/div&gt;').appendTo(buttonContainer);
     // Create the button that will start the tour again
-    var button = $($jsontool.serialize("&lt;button id='tourResume' class='btn btn-default btn-xs'&gt;$services.icon.renderHTML('info') #tr('tour.popover.show')&lt;/button&gt;")).appendTo(buttonContainer);
+    const button = $('&lt;button id=\'tourResume\' class=\'btn btn-default btn-xs\'&gt;&lt;/button&gt;')
+      .html("$escapetool.javascript($services.icon.renderHTML('info')) #tr('tour.popover.show')")
```

Comment:
Having a second pair of eyes checking that I did not miss anything related to escaping here wouldn't hurt, but looks good otherwise.
Thanks!